src/components/POLine/Item/ItemForm.js

```diff
@@ -271,7 +273,7 @@ class ItemForm extends Component {
                 disabled={isPostPendingOrder}
               />
               <div className={css.addButton}>
-                {this.selectInstanceModal(isPostPendingOrder)}
+                {this.selectInstanceModal(isSelectInstanceVisible)}
```

Comment:
isn't it more logical to write `{isSelectInstanceVisible && <InstancePlugin addInstance={this.onAddInstance} />}` ?